bin/utils/db_log.py

```diff
@@ -30,12 +30,13 @@
 from threading import Lock
 
 import click
-import sqlparse
 from pygments import highlight
 from pygments.formatters.terminal256 import Terminal256Formatter
 from pygments.lexers.agile import PythonLexer, PythonTracebackLexer
 from pygments.lexers.sql import SqlLexer
 
+import sqlparse
```

Comment:
sorry, forgot to mention that you need to install the dev requirements (`requirements.dev.txt`) as well